topics/appendix/tools/tools_gradle_intellij_plugin.md

```diff
@@ -110,6 +110,64 @@ pluginManagement {
 </tabs>
 
 
+## Contributing
+There are many ways to contribute to the Gradle IntelliJ Plugin project, and each of them is valuable to us.
+Every submitted feedback, issue, or pull request is highly appreciated.
+
+### Issue Tracker
+The Gradle IntelliJ Plugin is an Open-Source project hosted on GitHub: [JetBrains/gradle-intellij-plugin](https://github.com/JetBrains/gradle-intellij-plugin).
+All project sources, discussions, and issues are publicly available, as we rely on the [GitHub Issues](https://github.com/JetBrains/gradle-intellij-plugin/issues) to track any feature requests or reported problems.
+
+Before reporting an issue, please update your configuration to use always the [latest release](#usage), or try with the [snapshot release](#snapshot-release), which contains not-yet publicly available changes.
+
+If you find your problem unique, and it wasn't yet reported to us, [file an issue](https://github.com/JetBrains/gradle-intellij-plugin/issues/new) and provide as many details as possible, e.g.:
+- issue description
+- full stack trace
+- steps to reproduce it
+- plugin version you use
+- your build configuration
+- ideally, the minimal project will let us reproduce the issue locally.
+
+### Building Project
+If you want to run the Gradle IntelliJ Plugin directly from sources, i.e., to test it against the latest changes, debug the plugin, or verify your changes before creating a pull request, clone it from our GitHub repository:
+
+{style="narrow"}
+HTTPS
+: https://github.com/JetBrains/gradle-intellij-plugin.git
+
+SSH
+: git@github.com:JetBrains/gradle-intellij-plugin.git
+
+GitHub CLI
+: gh repo clone JetBrains/gradle-intellij-plugin
+
+There are three possible ways to run the Gradle IntelliJ Plugin using its sources:
```

Comment:
I think that after using a colon-ending sentence it should be a list or it should be just a regular sentence.